src/Data/Avro/Encoding/FromAvro.hs

```diff
@@ -315,22 +316,17 @@ getBlocksOf env t = do
     vs <- replicateM (fromIntegral blockLength) (getField env t)
     (vs:) <$> getBlocksOf env t
 
-writeByPositions :: MV.MVector s Value -> [(Int, Value)] -> ST s ()
-writeByPositions mv writes = foldl (>>) (return ()) (fmap (go mv) writes)
-  where go :: MV.MVector s Value ->  (Int, Value) -> ST s ()
-        go mv (n, v) = MV.write mv n v
-
 getRecord :: HashMap Schema.TypeName ReadSchema -> [ReadSchema.ReadField] -> Get (Vector Value)
 getRecord env fs = do
-  moos <- forM fs $ \f ->
```

Comment:
Also, what does `moos` mean?

Reply:
LOL, we both know the guy who names things `moo` when he can't come up with a better name (or is in a mood, or just doesn't care) :)
Must be remaining of his experimentations, or maybe me picking up the same practice :))))